### ISSUE TITLE
Changing resolution turns all text into black

### ISSUE THREAD

**Issue by Yihka** [tonihele/OpenKeeper#451]
1. `gradlew run`
2. Go to main screen
3. go to settings
4. Go to graphic settings
5. Change resolution to something else
6. Click the green 'V' button at the right bottom
7. You get the following, see screenshot

Going to other pages or opening stuff, everything stays black. Closing and opening saves the resolution and solves the problem.

![changing-resolution](https://github.com/tonihele/OpenKeeper/assets/7753711/a1b7869a-0eaa-49cf-a47d-b04f39b99a22)

**Comment by tonihele** [comment]
Yeah, it has to be fixed upsteam https://github.com/jMonkeyEngine/jmonkeyengine/issues/1013.
